File: src/Api/Models/DTOs/RegisterDTO.cs
using Api.Enums;

namespace Api.DTOs;
public class RegisterDTO
{
    public string Email {get;set;} = string.Empty;
    public string FirstName {get;set;} = string.Empty;
    public string LastName {get;set;} = string.Empty;
    public string Password {get;set;} = string.Empty;
    public string Username {get;set;} = string.Empty;
    public string PhoneNumber {get;set;} = string.Empty;
    public DateTime Birthdate {get;set;}
    public Gender Gender {get;set;} 
}

File: src/Client/src/models/dtos/RegisterDto.ts
export class RegisterDto 
{
    public email : string = '';
    public firstName : string = '';
    public lastName : string = '';
    public password : string = '';
    public username : string = '';
    public phoneNumber :string = '';
    public birthdate : Date = new Date();
    public gender : number = 0; 
}